pkg/types/powervs/platform.go

```diff
@@ -0,0 +1,57 @@
+package powervs
+
+// Platform stores all the global configuration that all machinesets
+// use.
+type Platform struct {
+
+	// ServiceInstanceID is the ID of the Power IAAS instance created from the IBM Cloud Catalog
+	ServiceInstanceID string `json:"serviceInstanceID"`
+
+	// PowerVSResourceGroup is the resource group for creating Power VS resources.
+	PowerVSResourceGroup string `json:"powervsResourceGroup"`
+
+	// Region specifies the IBM Cloud colo region where the cluster will be created.
+	Region string `json:"region"`
+
+	// Zone specifies the IBM Cloud colo region where the cluster will be created.
+	// At this time, only single-zone clusters are supported.
```

Comment:
"At this time" implies that at some point in the future we may support mutli-zone clusters. At that point, the `Zone` field will need to be deprecated in favor of a `Zones` field. Do you have a feel for (1) the likelihood of supporting multi-zone clusters and (2) the timeframe for said support?

Reply:
Good point, but no I don't. AFAIK it's not in the roadmap yet.